rfc/0002/README.md

```diff
@@ -0,0 +1,165 @@
+Rabbit storage for containerized applications
+=============================================
+
+**authors**: Nate Roiger <nate.roiger@hpe.com> |
+**state**: ideation
+
+---
+
+For Rabbit to provide storage to a containerized application there needs to be _some_ mechanism. The remainder of this RFC proposes that mechanism.
+
+Actors
+------
+
+There are several different actors involved
+
+- The AUTHOR of the containerized application
+- The ADMINISTRATOR who works with the author to determine the application requirements for execution
+- The USER who intends to to use the application using the 'container' directive in their job specification
+- The RABBIT software that interprets the #DWs and starts the container during execution of the job
+
+There are multiple relationships between the actors:
+- AUTHOR to ADMINISTRATOR: The author defines how their application is executed and the NNF storage requirements
+- Between the AUTHOR and USER: The application expects certain storage, and the #DW must meet those expectations.
+- ADMINISTRATOR to RABBIT: Admin tells Rabbit how to run the container
+- Between USER and RABBIT: User provides the #DW container directive in the job specification, and Rabbit validates & interprets the directive.
+
+Proposal
+--------
+
+The proposal below might take a couple of read-throughs; I've also added a concrete example afterward that might help.
+
+1. The AUTHOR writes their application expecting NNF Storage at specific locations. For each storage requirement, they define...
+    1. a unique name for the storage which can be referenced in the 'container' directive
+    2. the expected storage types; if necessary
+    3. the required mount path or mount path prefix
+    4. other constraints or storage requirements (i.e. minimum capacity)
+2. The AUTHOR works with the ADMINISTRATOR to define, a unique name for the program, the pod specification for executing their program, and the NNF storage requirements described above. The ADMINISTRATOR creates a corresponding _NNF Container Profile_ custom kubernetes resource.
```

Comment:
Maybe worth listing those 3 out in a list:
1. Unique name
2. pod spec
3. NNF storage requirements